### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by hannahbaeq**

### CODE REVIEW

File: starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java [hunk 1]
```diff
@@ -10,49 +11,46 @@
 import org.springframework.data.mongodb.core.index.IndexOperations;
 import org.springframework.util.Assert;
 
-/**
```
Comment (JackSkandalis) [inline starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java:13]: So Hannah, you actually want to leave these comments in the /** form. These are called Javadoc comments, and what they do is provide important information to a developer that might want to implement this method in the future. Here's a good description of what they are used for: https://www.baeldung.com/javadoc

File: starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java [hunk 2]
```diff
@@ -52,4 +48,41 @@ public void findAll() {
...
+    Inventory item2 = new Inventory();
+    item1.setName("item1");
+    item2.setName("item2");
+    item1.setProductType(PRODUCT_TYPE);
+    item2.setProductType(PRODUCT_TYPE);
+    // Call the function you want to test
+    this.inventoryDAO.create(item1);
+    this.inventoryDAO.create(item2);
+    // Make sure that the objects in Inventory == 2
+    Assert.assertEquals(2, this.mongoTemplate.findAll(Inventory.class).size());
```
Comment (elopezhaz) [inline starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java:66]: You should be able to combine these tests into one create test by just adding those extra asserts at the end here.

Comment (JackSkandalis) [inline starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java:66]: Use this.inventoryDAO.findAll() here just like you did with your second test.

File: starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java [hunk 3]
```diff
@@ -52,4 +48,41 @@ public void findAll() {
...
+  }
+
+  @Test
+  // Test for Create -> check names of item
+  public void create2() {
+    // Creating two new items to "create"
+    Inventory item1 = new Inventory();
+    Inventory item2 = new Inventory();
+    item1.setName("item1");
```
Comment (JackSkandalis) [inline starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java:75]: This is called a "Magic String", don't do this.

**Review by JackSkandalis — CHANGES_REQUESTED** [review]

**Pull request merged**